packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4019,11 +4019,30 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   public usePhoneAudio(phoneNumber: string) {
-    if (!phoneNumber) {
-      return this.dialInPstn();
-    }
+    const promise = phoneNumber ? this.dialOutPstn(phoneNumber) : this.dialInPstn();
+
+    return promise
+      .then(() => {
+        LoggerProxy.logger.info(
```

Comment:
2 reasons for which I kept them separate are:
1. It will be a very long message if I merge them in a single log which will affect user readability.
2. Both logs convey message action for 2 different methods(`dialout/in()` and `stopTracks()`) so it is recommended to keep them separate as per other logs in the same file.

Still if you feel it should be in a single log then I can change it to one.